Libplanet/Net/ITransport.cs

```diff
@@ -1,49 +1,162 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.Contracts;
 using System.Threading;
 using System.Threading.Tasks;
 using Libplanet.Net.Messages;
 
 namespace Libplanet.Net
 {
-    internal interface ITransport : IDisposable
+    /// <summary>
+    /// An interface to handle p2p, includes trading <see cref="Message"/>
```

Comment:
I suggested to use *peer-to-peer* instead of *p2p* because `Swarm<T>.BootstrapAsync(...)` used it.

```suggestion
    /// An interface to handle peer-to-peer networking, includes trading <see cref="Message"/>
```